docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -794,6 +794,18 @@ protocol()
 Returns the string representation of :repo:`HTTP protocol <envoy/http/protocol.h>`
 used by the current request. The possible values are: ``HTTP/1.0``, ``HTTP/1.1``, ``HTTP/2`` and ``HTTP/3*``.
 
+.. _config_http_filters_lua_stream_info_downstream_direct_local_address:
+
+downstreamDirectLocalAddress()
```

Comment:
```suggestion
``downstreamDirectLocalAddress()``
```

Reply:
I am not sure I understand this change, would you like me to do it for the whole file ?
I does not seems to be the case for the other method

Reply:
this just makes it a `literal` - ie its not an english word - its literally this

anywhere this is used in docs it should be made literal - with the exception of in link text as rst/sphinx does not currently handle literal links

Reply:
make sense, I replace everywhere where applicable in the file

Reply:
great, thanks